scenes/game/game.gd

```diff
@@ -625,7 +633,7 @@ func spawn_deck(deck_id,
 		var logic_card : GameCard = card_db.get_card(card.id)
 		var image_path = get_card_image_path(deck_id, logic_card)
 		var new_card = create_card(card.id, logic_card.definition, image_path, card_back_image, deck_card_zone, is_opponent)
-		if observer_mode:
+		if observer_mode and not replay_mode:
```

Comment:
Is this enough to see the opponent cards too? I don't think it is?

Reply:
It isn't, I don't think this would be good to do. The primary purpose of replays is to review decisions that were made in a given game state, with the limited information that was available at the time. Showing the opponent's hand would inevitably pollute that assessment, even if we try not to take it into account.

Reply:
Maybe we should make this an option?
I could see wanting to see both hands if you're casting the game or something.
Or if you want to know why your opponent played what they did at the time.